resources/js/Pages/Home.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+    <Layout>
+    <p class="text-black-15 text-xl bg-primary sm:bg-black-15">Heading text</p>
+    </Layout>
```

Comment:
does your IDE support code format? make sure it automatically does the format. Please remove this sample text and create a new prop for inject
```suggestion
    <Layout>
        <p class="text-black-15 text-xl bg-primary sm:bg-black-15">Heading text</p>
    </Layout>
```